### PR TITLE
Implement SSM documents resource

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### :rocket: Added
+* Added `aws_ssm_documents` resource
+* Added `aws_ssm_instances` resource
```
Comment (roneli) [inline CHANGELOG.md:11]: link PRs?

**Review by roneli — APPROVED** [review]
LGTM, @zagronitay can you review the TF files?

File: CHANGELOG.md (hunk 1 above)
Comment (irmatov) [reply CHANGELOG.md]: Linked

File: resources/integration_tests/aws_ssm_documents_test.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+				Data: map[string]interface{}{
+					"description":     "Check ip configuration of a Linux instance.",
+					"document_format": "JSON",
+					"document_type":   "Command",
+					"name":            fmt.Sprintf("%sdoc%s", res.Prefix, res.Suffix),
+					"account_ids":     []interface{}{"all"},
+					"tags": map[string]interface{}{
+						"TestId": res.Suffix,
+						"Type":   "integration_test",
+					},
```
Comment (zagronitay) [inline resources/integration_tests/aws_ssm_documents_test.go:27]: Can you add the name to tags as well? WDYT?

File: resources/integration_tests/infra/aws_ssm_documents.tf [hunk 3]
```diff
@@ -0,0 +1,29 @@
+resource "aws_ssm_document" "mytestdoc" {
```
Comment (zagronitay) [inline resources/integration_tests/infra/aws_ssm_documents.tf:1]: lets change the label to "ssm_document"

**Review by zagronitay — CHANGES_REQUESTED** [review]
Minor changes

File: resources/integration_tests/aws_ssm_documents_test.go (hunk 2 above)
Comment (irmatov) [reply resources/integration_tests/aws_ssm_documents_test.go:27]: document's name is listed already slightly above, isn't it? Or do you want to add some new arbitrary tag?

File: resources/integration_tests/infra/aws_ssm_documents.tf (hunk 3 above)
Comment (irmatov) [reply resources/integration_tests/infra/aws_ssm_documents.tf:1]: Done

**Pull request merged**